community.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: Community
+description: How to contribute to the Clowder community.
+background: '/PATH_TO_IMAGE'
+---
+
+There are many ways to get involved in the Clowder community. Below we have identified a few ways and pointers to get started. *The easiest way to get more information is to join us on [Slack][slack].* Before participating, please take a momente to [read our Code of Conduct](https://github.com/clowder-framework/clowder/blob/develop/CODE_OF_CONDUCT.md).
```

Comment:
missing url for slack

Reply:
not sure never seen the reference to footnotes.

Reply:
yeah, it's in the footnote. Was thinking of linking to slack everywhere.